### DISCUSSION TITLE
Velociraptor 0.6.7 Org Issue

### DISCUSSION THREAD

**Discussion by TerryG1967** [Velocidex/velociraptor#2353] [Q&A]
HI Guys,

I am trying to use velociraptor on windows and create an org to store some clients in. when i run the server artifact Server.Orgs.NewOrg i get the message below even though i am signed in as admin.

State ERROR
Error Symbol user_create not found. Current Scope is: [NULL], [$cache, $device_manager, config, $acl, $uploader, Artifact, $root], [OrgName], [org_record], [Server_Orgs_NewOrg_0_1], [$Query], []

I am running version 0.6.7, any help would be gratefully received.

Thanks

Terry

**Comment by scudette** [comment]
You can actually use the GUI to add orgs in 0.6.7-4 now so that's probably the easiest way to manage users and orgs

**Reply by TerryG1967** [reply]
HI There

I have the version you suggested but still i cant seem to add orgs . I maybe missing something here but do you have any details of how i go about adding a new org.

Terry

**Reply by scudette** [reply]
Click on the user management sidebar icon then select a user ( if the user you need to be admin does not exist yet, you can add it first). On the second part of the UI that shows the orgs the user belong to, click the + icon to add a new org. 

The user will become admin on the new org. You can also add other users by selecting their roles in that org.


The original question shows the error that user_create is not defined. But this is a standard function in 0.6.7-4

https://docs.velociraptor.app/vql_reference/server/user_create/

It might be that during the config generation setup you opted to block potentially dangerous vql functions on the server. This adds an allow list to the config file. If you have that allow list you can add that vql function to the list and restart the server.

**Comment by TerryG1967** [comment]
Thanks for this update. i can now create orgs using server artifacts Server.Orgs.NewOrg, but still no option to add a new org on the gui (which isnt a real issue now)

I have tried downloading the client config for the new org and using the new yaml file instead of the original client.config.yaml on a client pc already connected to the server but when i try and a hunt on the machine in the new org it doesn't display the hunt and the screen is blank as if the hunt had not been ran

i thought that when i associated this new config it will check the nonce in the file and associate this with the relevant org.

i have multiple customers that i will be trialing this with and want to be able to just run analysis on a certain org... 

Thanks for any help you can provide on this .

Terry
